packages/cerebral/package.json

```diff
@@ -18,7 +18,8 @@
   "homepage": "http://cerebral.github.io/cerebral-website",
   "dependencies": {
     "eventemitter3": "^2.0.2",
-    "function-tree": "^0.5.0-b"
+    "function-tree": "^0.5.0-b",
+    "prop-types": "^15.5.4"
```

Comment:
This should not be there. React is not a hard dependency to Cerebral, you can also use inferno... and we are keeping options open to use other views as well :)